services/QuillLMS/spec/models/app_setting_spec.rb

```diff
@@ -22,6 +22,25 @@
 RSpec.describe AppSetting, type: :model do
   let(:user) { create(:user) }
 
+  describe '#logged_out_enabled?' do
+    let!(:app_setting) do
+      create(:app_setting, name: 'egosum', enabled: true, percent_active: 100)
```

Comment:
The repeated use of `'egosum'` in this spec suggests to me that it's probably worth extracting into `let(:name) { 'egosum' }` to avoid risks around typo-ing a repeated constant.

That would also let you use: `subject { AppSetting.logged_out_enabled?(name:) }` in all of the tests.